src/Dfc.ProviderPortal.TribalExporter/Functions/ApprenticeshipDeltaExport.cs

```diff
@@ -14,6 +14,7 @@ namespace Dfc.ProviderPortal.TribalExporter.Functions
 {
     public static class ApprenticeshipDeltaExport
     {
+        [Disable]
```

Comment:
Hey Jav, do you need the functions disabled?

Reply:
good point ! no they need to be enabled
reject this one, I will request another one !